controllers/home-routes.js

```diff
@@ -69,16 +69,18 @@ router.get("/dashboard", withAuth, async (req, res) => {
     try {
         const dbHomeData = await Home.findAll({
             where: {
-                user_id: req.session.user_id,
```

Comment:
we def don't want to change this.  We need the req.session info for the site to work.